vispy/geometry/tests/test_meshdata.py

```diff
@@ -31,4 +32,50 @@ def test_meshdata():
     assert_array_equal(square_edges, mesh.get_edges())
 
 
+def test_face_normals():
+    dtype = np.float32
+    vertices = np.array([
+        (0, 0, 0),
+        (1, 0, 0),
+        (0, 1, 0),
+        (0, 0, 1),
+    ], dtype=dtype)
+    faces = np.array([(0, 1, 2), (0, 2, 3)], dtype=int)
+
+    expected_normals = np.array([(0, 0, 1), (1, 0, 0)], dtype=dtype)
+
+    computed_normals1 = meshdata._compute_face_normals(vertices, faces)
```

Comment:
Could you use the public methods (`get_x_normals`) so we're testing the actual MeshData interfaces instead of the implementation of how it is done. Shouldn't they return the same thing?

Reply:
This is already tested in `test_meshdata()`.